src/app/shared/model/distribution.model.ts

```diff
@@ -86,6 +86,13 @@ export class DistributionModel extends BaseModel {
     return {podcast, versionTemplates};
   }
 
+  get feederPodcastId(): string {
+    if (this.kind === 'podcast' && this.url) {
+      const parts = this.url.match(/\/(.[^\/]*)$/);
```

Comment:
This regex is a bit hard to follow.  Maybe more like a `this.url.split('/').pop()` or something?